sdk/lib/opentelemetry/sdk/configurator.rb

```diff
@@ -143,41 +143,66 @@ def install_instrumentation
       end
 
       def configure_span_processors
-        processors = @span_processors.empty? ? [default_span_processor] : @span_processors
+        processors = @span_processors.empty? ? [wrapped_exporter_from_env].compact : @span_processors
         processors.each { |p| tracer_provider.add_span_processor(p) }
       end
 
-      def default_span_processor
-        Trace::Export::SimpleSpanProcessor.new(
-          Trace::Export::ConsoleSpanExporter.new
-        )
+      def wrapped_exporter_from_env
+        exporter = ENV.fetch('OTEL_TRACES_EXPORTER', 'otlp')
+        case exporter
+        when 'none' then nil
+        when 'otlp' then fetch_exporter(exporter, 'OpenTelemetry::Exporter::OTLP::Exporter')
+        when 'jaeger' then fetch_exporter(exporter, 'OpenTelemetry::Exporter::Jaeger::CollectorExporter')
+        when 'zipkin' then fetch_exporter(exporter, 'OpenTelemetry::Exporter::Zipkin::Exporter')
+        else
+          OpenTelemetry.logger.warn "The #{exporter} exporter is unknown and cannot be configured, spans will not be exported"
+          nil
```

Comment:
This (`[wrapped_exporter_from_env].compact` and returning `nil` in the default cases) means we end up with the default NoopSpanProcessor rather than adding another one.